Formula/sysbench.rb

```diff
@@ -30,7 +32,7 @@ def install
     # https://github.com/LuaJIT/LuaJIT/issues/518: set to 10.14 to build on Catalina.
     ENV["MACOSX_DEPLOYMENT_TARGET"] = (DevelopmentTools.clang_build_version >= 1100) ? "10.14" : MacOS.version
```

Comment:
Since we're using a Homebrew LuaJIT, I think this line (and the comment block before it) can be removed.